src/gui/src/layoutViewer.cpp

```diff
@@ -3426,6 +3443,8 @@ void LayoutViewer::addMenuAndActions()
       = highlight_menu->addAction(tr("Input Nets"));
   menu_actions_[HIGHLIGHT_ALL_NETS_ACT]
       = highlight_menu->addAction(tr("All Nets"));
+  menu_actions_[HIGHLIGHT_ALL_BUFFER_TREES_ACT]
+      = highlight_menu->addAction(tr("All buffer trees"));
```

Comment:
@maliberty is there value in being able to select just the input trees vs. output trees?

Reply:
As far I understand the buffer Tree object is that it find all the buffer in the fan out and facing of the specific net until it reach to none buffer inst.
So in order to choose only output or input I need to add support in buffer tree object, is that OK?